### DISCUSSION TITLE
How to host the bot for free?

### DISCUSSION THREAD

**Discussion by nuexq** [stegripe/rawon#1428] [Quest and Answer]
how i can host the bot, I'm a front-end developer and have experience with js and ts, I try to host the bot on replit but when I run a song it stopped quickly is there a solution or what a hosting platform for free

**Comment by Mednoob** [comment]
What stream strategy did you use, and was there any error in the console?

**Reply by nuexq** [reply]
I doesn't undrstand i use just replit and uptimerobot and when I run any song it stopped and nothing show in the console just log of the song start and the song stop, I try today render.com but it say error about yt-dlb and he couldn't find it
